### PR TITLE
revert(splits): use random split to resolve distribution drift (#131)

### LINKED ISSUE #131: Updated results for distribution drift issue
There is a significant distribution drift due to the new training split we had created to exclude any proteins in OncoKB...
- This is evident by the **drastic difference in performance when training with the split vs when training on a random split.**
- Most evident in the PDBbind dataset.
- [ ] get distributional stats to provide additional support for this claim
   - Via sequence identity?

## TODOs:
- [x] build new dataset with a random, but unified test set (any overlaping proteins across datasets that are in the test set, must also be in the test set).
- [x] Identify the list of ligand-protein pairs from oncoKB that are not in the training set, which can be used for Mutagenesis analysis.
- [ ] profit??

### PR DESCRIPTION
**Pull request by jyaacoub**
resolves #131

### CODE REVIEW

**Review by jyaacoub — COMMENTED** [review]
lgtm
